product_docs/docs/pem/9/pem_rel_notes/920_rel_notes.mdx

```diff
@@ -6,26 +6,26 @@ New features, enhancements, bug fixes, and other changes in PEM 9.2.0 include:
 
 | Type        | Description                                                                                      |
 | ----------- | -------------------------------------------------------------------------------------------------|
-| Enhancement | The default sample end time for performance diagnostics is now `Now` rather than the time the window was first opened, meaning the data will update upon clicking 'Run'. |
-| Bug fix     | Fixed the `Replication Time Lag` graph status issue on the Streaming Analysis dashboard. This fix displays the status as stopped if the server is down/offline. [Support ticket: #91220] |
-| Bug fix     | Removed the agent status from Failover Manager Dashboard under `efm_cluster_node_status`. [Support ticket: #1228607] |
+| Enhancement | The default sample end time for performance diagnostics is now **Now** rather than the time the window was first opened. The data now updates when you click **Run**. |
```

Comment:
```suggestion
| Enhancement | The default sample end time for performance diagnostics is now **Now** rather than the time the window was first opened. The data now updates when you select **Run**. |
```